packages/uniforms-bootstrap4/src/ErrorField.js

```diff
@@ -6,8 +6,8 @@ import nothing        from 'uniforms/nothing';
 
 const Error = ({children, className, error, errorMessage, ...props}) =>
     !error ? nothing : (
-        <div className={classnames('card', className)} {...filterDOMProps(props)}>
-            <div className="card-block">
+        <div className={classnames('card', 'mb-3', className)} {...filterDOMProps(props)}>
```

Comment:
Is this `mb-3` part really needed? We shouldn't enforce your layout.

It's also a few times more below.

Reply:
At version `4.0.0` `margin-bottom` was removed from some components. This `mb-3` class is added here to provide proper display the demo page. Without this submit buttons etc are stick to the form what imho looks bad.